app/forms/field_handlers/number_handler.py

```diff
@@ -0,0 +1,118 @@
+from typing import Union
+
+from wtforms import IntegerField, DecimalField
+
+from app.data_model.answer_store import AnswerStore
+from app.forms.fields.custom_decimal_field import CustomDecimalField
+from app.forms.fields.custom_integer_field import CustomIntegerField
+from app.forms.field_handlers.field_handler import FieldHandler
+from app.questionnaire.location import Location
+from app.validation.validators import NumberCheck, NumberRange, DecimalPlaces
+
+
+class NumberHandler(FieldHandler):
+    MANDATORY_MESSAGE = 'MANDATORY_TEXTFIELD'
+    MAX_NUMBER = 9999999999
+    MIN_NUMBER = -999999999
+    MAX_DECIMAL_PLACES = 6
```

Comment:
This doesn't seem to be used anywhere but a test?.

Reply:
Should be used in get_number_field_dependencies